contrib/kpatch.spec

```diff
@@ -70,7 +68,7 @@ make %{_smp_mflags}
 %install
 rm -rf %{buildroot}
 
-make install DESTDIR=%{buildroot}
+PREFIX=/usr make install DESTDIR=%{buildroot}
```

Comment:
Probably should be `PREFIX=/%{_usr}`.  Also, putting it next to DESTDIR (or putting DESTDIR next to PREFIX) would make it a little more readable.